go.mod

```diff
@@ -2,103 +2,206 @@ module github.com/erh/viamrtsp
 
 go 1.21
 
-toolchain go1.21.8
-
 require (
 	github.com/bluenviron/gortsplib/v4 v4.8.0
```

Comment:
Resolved some conflicts from main from the deps bump PR. Double checked that it looks fine but want to make sure

Reply:
We need to include linter deps. I included them and re-ran `go mod tidy`, the purpose of the original comment in this thread is to call out to double check that I properly resolved the conflicts from rebasing the upgrade deps changes from main i.e. did not overwrite the upgraded deps with the older versions. Can ignore the addition of the new linter deps and evaluated indirect deps